tests/fxl-page-spread-break/EPUB/package.opf

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" xmlns:epub="http://www.idpf.org/2007/ops" version="3.0" xml:lang="en" unique-identifier="pub-id">
+  <metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+    <dc:coverage>Layout Rendering</dc:coverage>
```

Comment:
In say https://github.com/w3c/epub-tests/blob/main/tests/fxl-page-spread-center/EPUB/package.opf, we have called this Fixed Layout, and there is no existing Layout Rendering table at https://w3c.github.io/epub-tests/.

Are you doing this because several of the properties in the FXL section of the spec aren't really FXL-exclusive? If so, I am open to a change, but I think we should also
1) consider a spec change (just renaming?)
2) change this in all other tests (in a separate PR)
3) and rename the fxl- prefix too (in the same PR as the above)

So for now I think this should be Fixed Layout, unless I misunderstand.

Reply:
I am thinking of option 2, this is a group of tests under section 8, Layout Rendering, but I need to see which tests fall into here and which don't, will do in a seperate PR, but I'll change this one to FXL for now.